include/roboptim/core/function.hh

```diff
@@ -29,9 +29,9 @@
 # define EIGEN_YES_I_KNOW_SPARSE_MODULE_IS_NOT_STABLE_YET
 
 # include <roboptim/core/alloc.hh>
-# include <Eigen/Core>
-# include <Eigen/Dense>
-# include <Eigen/Sparse>
+# include <eigen3/Eigen/Core>
+# include <eigen3/Eigen/Dense>
+# include <eigen3/Eigen/Sparse>
```

Comment:
You still have extras `eigen3/` here and in other files 😅

Reply:
Sorrrry. Fixed in  ca16c3e.
While being at it, fixed warnings.